### ISSUE TITLE
argv misgenerated with quoted whitespace parameters

### ISSUE THREAD

**Issue by ksherlock** [byteworksinc/ORCALib#3]
`C_StartUp` scans the command line twice, once to count the number of arguments, then again to generate the argv vector (after allocating memory)

The argument counting code treats a double-quote as a whitespace, so something like `""` or `"  "` doesn't increment argc. The second pass does handle it correctly and the argv vector is corrupted due to expected vs actual size mismatch.

https://github.com/byteworksinc/ORCALib/blob/master/cc.asm#L196-L219

**Comment by ksherlock** [comment]
Reproduce: 
```
#include <stdio.h>
#include <orca.h>

int main(int argc, char **argv) {
  int i;
  printf("%s\n", commandline());
  for (i = 0; i < argc; ++i) printf("%d: %s\n", i, argv[i]);
  return 0;
}
```

```
#params "   " hello
params "   " hello
0: 
1:    
2: hello
```
argv handle:

```
021600:	0c 16 02 00 14 16 02 00  19 16 02 00 00 00 00 00 	................
021610:	6d 73 00 22 20 20 20 00  20 68 65 6c 6c 6f 00 00 	ms."   . hello..
```

```
#params " x " hello 
params " x " hello
0: params
1:  x 
2: hello
```

argv handle:
```
021600:	10 16 02 00 18 16 02 00  1d 16 02 00 00 00 00 00 	................
021610:	70 61 72 61 6d 73 00 22  20 78 20 00 20 68 65 6c 	params." x . hel
021620:	6c 6f 00 00 00 00 00 00  00 00 00 00 00 00 00 00 	lo..............
```

**Comment by ksherlock** [comment]
not yet tested, but something like this should count parameters correctly:

```
* skip whitespace
lb2	lda [cLine],y
	beq lb6
	iny 
	cmp #' '
	beq lb2
	cmp #TAB
	beq lb2
	inx ; argc++
	cmp #'"'
	beq lb5

* advance to next whitespace
lb4	lda [cLine],y 
	beq lb6
	iny
	cmp #' '
	beq lb2
	cmp #TAB
	beq lb2
	bra lb4

* advance to next "
lb5	lda [cLine],y
	beq lb6
	iny
	cmp #'"'
	beq lb2
	bra lb5
```

**Comment by sheumann** [comment]
Fixed in commit d95f9f29d.
